test/db/tools/rz_bin

```diff
@@ -1084,143 +1084,128 @@ CMDS=<<EOF
-            "member_name": "c",
-            "offset": 0
-          }
-        ]
+        "member_type": "char [4]",
+        "member_name": "a",
+        "offset": 0
       },
       {
-        "type": "structure",
-        "name": "union_typedef",
-        "size": 4,
-        "members": [
-          {
-            "member_type": "char",
-            "member_name": "a",
-            "offset": 0
-          },
-          {
-            "member_type": "uint16_t",
-            "member_name": "b",
-            "offset": 0
-          },
-          {
-            "member_type": "int32_t",
-            "member_name": "c",
-            "offset": 0
-          }
-        ]
+        "member_type": "int32_t",
+        "member_name": "b",
+        "offset": 0
       }
-    ],
-    "gvars": [
+    ]
+  },
+  {
+    "type": "structure",
+    "name": "unnamed_member_types_typedef",
+    "size": 8,
+    "members": [
       {
-        "address": 12292,
-        "symtype": 0,
-        "section_name": ".data",
-        "gdata_name": "_uninitialized_global"
+        "member_type": "union <unnamed-tag>",
+        "member_name": "a",
+        "offset": 0
       },
       {
-        "address": 12288,
-        "symtype": 0,
-        "section_name": ".data",
-        "gdata_name": "_initialized_global"
-      },
+        "member_type": "int32_t",
+        "member_name": "b",
+        "offset": 4
+      }
+    ]
+  },
+  {
+    "type": "structure",
+    "name": "bitfield_typedef",
+    "size": 1,
+    "members": [
+      
+    ]
+  },
+  {
+    "type": "union",
+    "name": "union_typedef",
+    "size": 4,
+    "members": [
       {
-        "address": 4096,
-        "symtype": 2,
-        "section_name": ".text",
-        "gdata_name": "_function"
+        "member_type": "char",
+        "member_name": "a",
+        "offset": 0
       },
       {
-        "address": 4182,
-        "symtype": 2,
-        "section_name": ".text",
-        "gdata_name": "_mainCRTStartup"
+        "member_type": "int16_t",
+        "member_name": "b",
+        "offset": 0
       },
       {
-        "address": 4120,
-        "symtype": 2,
-        "section_name": ".text",
-        "gdata_name": "_main"
+        "member_type": "int32_t",
+        "member_name": "c",
+        "offset": 0
       }
     ]
   }
+]{
+  "gvars": [
+    {
+      "address": 12292,
+      "symtype": 0,
+      "section_name": ".data",
+      "gdata_name": "_uninitialized_global"
+    },
+    {
+      "address": 12288,
+      "symtype": 0,
+      "section_name": ".data",
+      "gdata_name": "_initialized_global"
+    },
+    {
+      "address": 4096,
+      "symtype": 2,
+      "section_name": ".text",
+      "gdata_name": "_function"
+    },
+    {
+      "address": 4182,
+      "symtype": 2,
+      "section_name": ".text",
+      "gdata_name": "_mainCRTStartup"
+    },
+    {
+      "address": 4120,
+      "symtype": 2,
+      "section_name": ".text",
+      "gdata_name": "_main"
+    }
+  ]
+}{
+  "pdb": 
```

Comment:
this does not seem like valid json.

Reply:
These are 3 different JSON strings but pretty printed.

Reply:
What do you mean? THe line above this is `}{` and that is never valid json. The output of `rz-bin -Pj ${RZ_FILE}` should be valid json, otherwise something is wrong.